frontend/src/admin/components/user-list/user-list.tsx

```diff
@@ -1,17 +1,16 @@
-import React, {useState} from "react";
-import Users from "../../users.json";
+import {useState} from "react";
 import {User} from "../../models/user";
+import DropdownMenu from "../sidebar/dropdown/dropdown-menu";
 import UserListItem from "./user-list-item";
-import IconButton from "./icon-button";
-import "./user-list.scss";
```

Comment:
Why is the scss import removed? If it is not necessary, then please remove the corresponding scss file as well :)

Reply:
✅